packages/SystemUI/src/com/android/keyguard/KeyguardUpdateMonitorCallback.java

```diff
@@ -319,4 +319,8 @@ public void onLogoutEnabledChanged() { }
      */
     public void onBiometricsCleared() { }
 
+    /**
+     * Called when a pulsing is received.
+     */
+    public void onPulsing(boolean pulsing) { }
```

Comment:
why empty function?

Reply:
rest of the PR seems fine. I only can't understand this one

Reply:
That what I'm wondering too, empty like an abstract ones except it's not.
But looks like it will gets overrided just like regular lifecycle method

Reply:
Gets overrided withouth @Override?